services/blkmgr/interface.go

```diff
@@ -1,16 +1,27 @@
-package blockchain
+/*
+ * Copyright (c) 2020.
+ * Project:qitmeer
+ * File:interface.go
+ * Date:8/21/20 3:47 PM
+ * Author:Jin
+ * Email:lochjin@gmail.com
+ */
+
```

Comment:
Please keep using the standard qitmeer copyright declaration when you add copyright to a new source file.
```
// Copyright (c) 2020 The qitmeer developers
// Use of this source code is governed by an ISC
// license that can be found in the LICENSE file.
```